tests/store_test.go

```diff
@@ -111,6 +112,64 @@ func (s *StoreSuite) TestStoreFindOneReturnValues() {
 	s.resultOrError(doc, err)
 }
 
+func (s *StoreSuite) TestStoreFindAllReturnValues() {
```

Comment:
This seems to be unrelated to the PR. What does it test for?

Reply:
Yes, this is unrelated.
Now, FindAll() is okay about leaking, but in the future?
I intend to detect leaking by test for some basic functions exposed by generated store.